lnd.go

```diff
@@ -93,10 +93,29 @@ var (
 	}
 )
 
+// ListenerCfg is a wrapper around custom listeners that can be passed to lnd
+// when calling its main method.
+type ListenerCfg struct {
+	// WalletUnlocker can be set to the listener to use for the wallet
+	// unlocker. If nil a regular network listener will be created.
+	WalletUnlocker net.Listener
+
+	// RPCListener can be set to the listener to use for the RPC server. If
+	// nil a regular network listener will be created.
+	RPCListener net.Listener
+}
+
+// rpcListeners us a function type used for closures that fetches a set of RPC
```

Comment:
nit: `s/rpcListeners us/rpcListeners is`

Reply:
Fixed: https://github.com/lightningnetwork/lnd/pull/3282/commits/a641da4d9571fde5684f3b378e3ee3963bfc16e8